logger/google.go

```diff
@@ -0,0 +1,122 @@
+package logger
+
+import (
+	"encoding/json"
+
+	"cloud.google.com/go/logging"
+	"github.com/golang/protobuf/ptypes"
+	logtypepb "google.golang.org/genproto/googleapis/logging/type"
+)
+
+// Google logs to the Google cloud console logging platform.
+type Google struct {
+	// BaseEntry is used to define parameters other than Severity and Payload on the Google Logger.
+	BaseEntry logging.Entry
+	BlackList []Severity
+
+	// Logger here is what is actually doing the logging work. This interface is primarily for testing purposes.
+	// For example, we need to make sure that the logger discards all debug logs.
+	Logger interface {
+		Log(entry logging.Entry)
+	}
+}
+
+// NewGoogle returns a new Google logger using the provided client and name.
+func NewGoogle(client *logging.Client, name string) *Google {
+	return &Google{
+		Logger: client.Logger(name),
+	}
+}
+
+// WithBaseEntry populates the base entry for the google log, which will be used for all google logs.
+func (l *Google) WithBaseEntry(b logging.Entry) *Google {
+	return &Google{
+		BaseEntry: b,
+		Logger:    l.Logger,
+		BlackList: l.BlackList,
+	}
+}
+
+func (l *Google) WithBlacklist(s ...Severity) *Google {
```

Comment:
Need to avoid using "Blacklist" use block or deny wording.

Reply:
AllowList and BlockList are good alternatives.